README.md

```diff
@@ -2,10 +2,42 @@
 
 [![CircleCI](https://circleci.com/gh/Khan/wonder-blocks.svg?style=svg)](https://circleci.com/gh/Khan/wonder-blocks) [![codecov](https://codecov.io/gh/Khan/wonder-blocks/branch/master/graph/badge.svg)](https://codecov.io/gh/Khan/wonder-blocks)
 
-The Khan Academy Wonder Blocks design system. This is work-in-progress and a lot of things are still in motion.
+The Khan Academy Wonder Blocks design system. This is work-in-progress and a lot
+of things are still in motion.
 
 More information: https://wonder-blocks.netlify.com/
 
+## Getting started
+
+### Prerequisites
+
+- [Node.JS v8](https://nodejs.org/download/release/v8.16.1/)
+- [Yarn](https://yarnpkg.com/lang/en/docs/install/)
+
+### Installation
+
+To install Wonder Blocks, you need to run the following commands:
+
+#### `yarn install`
+
+Installs project dependencies
+
+#### `yarn start`
+
+Runs the docs in dev mode
+
+Now you can open http://localhost:6060 to view the docs. This page will
+automatically update as you make changes to components.
+
 ## Contributing
```

Comment:
I think we should add an explicit statement about accepting external contributions that makes it clear if we do or do not, and if we do, under what circumstances (i.e. what types of changes).

Reply:
That's a great suggestion, I'll add that shortly.